apps/src/dom.js

```diff
@@ -30,7 +30,10 @@ var addEvent = function(element, eventName, handler) {
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
     element.addEventListener(touchEvent, function(e) {
-      e.preventDefault();  // Stop mouse events.
+      // Stop mouse events and suppress default event handler to prevent
+      // unintentional double-clicking
+      e.preventDefault();
+      element.removeEventListener(eventName, handler);
```

Comment:
My understanding of the intent of this function is that we care about at least one of them happening; because methods for detecting which one we expect to happen are unreliable at best, we bind to both just in case